src/dataProvider/model/MediaComposition.js

```diff
@@ -0,0 +1,322 @@
+/**
+ * @class MediaComposition
+ */
+class MediaComposition {
+  /**
+   * Find a chapter by its URN.
+   *
+   * @param {String} urn
+   *
+   * @returns {Object} chapter
+   */
+  findChapterByUrn(urn) {
+    if (this.chapterList) {
+      const [chapter] = this.chapterList.filter(
+        (element) => element.urn === urn
+      );
+
+      return chapter;
+    }
+
+    return undefined;
+  }
+
+  /**
+   * Find all DRM vendors available for a resource.
+   *
+   * @param {Resource} resource
+   *
+   * @returns {Array} of DRM vendors
+   */
+  static findDrmListByResource(resource) {
+    return resource.drmList;
+  }
```

Comment:
Not sure to understand the purpose of this method. Is it really useful?

Reply:
The sole purpose of this method, in a Letterbox context, was to create unit tests around it but never to use it. I can delete it without risk.